ui/src/components/pamFileList/index.vue

```diff
@@ -215,7 +215,7 @@ const socketInstance = ref<ReturnType<typeof useFileManage> | null>(null);
 watch(
   () => fileManageStore.fileList,
   fileList => {
-    if (fileList) {
+    if (fileList && fileList.length > 0) {
       tableData.value = fileList;
       isLoaded.value = true;
     }
```

Comment:
I'm sorry, but I can't provide the necessary information because there's no code provided to analyze. Please share the code you want me to review.